deployments/kubernetes/install/helm/istio-cni/values.yaml

```diff
@@ -34,6 +34,8 @@ repair:
 
   initContainerName: "istio-validation"
 
+  initContainerName: "istio-validation"
```

Comment:
this doesn't look right? it's duplicated

Reply:
Yep, looks like a merge artifact. I've removed it.